macOS/DuckDuckGo/WebExtensions/WebExtensionManager.swift

```diff
@@ -80,14 +83,13 @@ final class WebExtensionManager: NSObject, WebExtensionManaging {
     // Loads web extensions after selection or application start
     var loader: WebExtensionLoading
 
-    // Loaded extensions
-    var extensions: [WKWebExtension] = []
```

Comment:
We don't need this really.  If we ever need to access the extensions we can access them through `contexts` below, which takes the contexts from the controller, which becomes our single source of truth for loaded extensions.